src/d3a/models/strategy/area_agents/two_sided_pay_as_bid_engine.py

```diff
@@ -91,9 +91,9 @@ def tick(self, *, area):
 
         for bid_id, bid in self.markets.source.get_bids().items():
             if bid.id not in self.bid_age:
-                self.bid_age[bid.id] = area.current_tick
+                self.bid_age[bid.id] = area.current_tick_in_slot
```

Comment:
This is not the current tick in the slot actually. IIRC this is the current tick for the whole simulation, therefore it never resets to 0.

Reply:
yes, this is true. This due to autorefactoring of Pycharm probably.
I wonder why this does not lead to a failing test.
BUT: I found out, that the "clock" of a market can only run for one slot_length, this is why, the market clock has to be updated with the current_tick_in_slot.

Reply:
The latter indeed is correct, there is no need to send the tick number for the whole simulation, tick in slot should suffice.